src/libPMacc/CMakeLists.txt

```diff
@@ -23,24 +23,25 @@
 ################################################################################
 # PMacc tests
 ################################################################################
+
 cmake_minimum_required(VERSION 3.3.0)
 project(PMaccTest)
 
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
hm, if you actually *have* a `PMaccTest` project, that one *should* be C++11 but ok-ish...
did you get problems leaving that in?